.clippy_allowed_lints

```diff
@@ -0,0 +1,39 @@
+# This is a list of allowed Clippy rules for the purposes of gradual migration.
+# Unfortunately, Clippy doesn't support allowing lints in a configuration file, so we roll our own.
+# See https://doc.rust-lang.org/stable/clippy/configuration.html
```

Comment:
Suggestion, putting these in `.cargo/config.toml`, instead, so that they are picked up from Rust Analyzer from `settings.json`:

```json
     "rust-analyzer.check.command": "clippy",
```

Reply:
Done in bf1bba4d9a73f607ea86a3d5a1ff53f657533b03